packages/compass-collection/src/modules/collection-tab.ts

```diff
@@ -4,22 +4,22 @@ import type Collection from 'mongodb-collection-model';
 import type { ThunkAction } from 'redux-thunk';
 import type AppRegistry from 'hadron-app-registry';
 import type { DataService } from 'mongodb-data-service';
+import type { workspacesServiceLocator } from '@mongodb-js/compass-workspaces/provider';
+import type { CollectionSubtab } from '@mongodb-js/compass-workspaces';
 
-type CollectionThunkAction<
-  ReturnType,
-  Action extends AnyAction = AnyAction
-> = ThunkAction<
-  ReturnType,
+type CollectionThunkAction<R, A extends AnyAction = AnyAction> = ThunkAction<
+  R,
   CollectionState,
   {
-    globalAppRegistry: AppRegistry;
     localAppRegistry: AppRegistry;
     dataService: DataService;
+    workspaces: ReturnType<typeof workspacesServiceLocator>;
   },
-  Action
+  A
 >;
 
 export type CollectionState = {
+  workspaceTabId: string;
```

Comment:
Do we have a ticket for changing this to use context value? Would be good to have it here in a TODO

Reply:
will link it! thanks!